server/osx/server_osx.py

```diff
@@ -331,28 +336,13 @@ def get_context():
        at least include title and executable.'''
 
     window_id, window_title = get_active_window()
-    if window_id is None:
-        return {}
-
-    cmd = '''tell application "System Events" to tell application process "%s"
-        try
-            get properties of window 1
-        on error errmess
-            log errmess
-        end try
-    end tell
-    ''' % window_id
-    script = applescript.AppleScript(cmd)
-    properties = {}
-    props = script.run()
-
-    properties = map_window_properties(props)
+    properties = get_window_properties(window_id)
     properties['id'] = window_id
     properties['title'] = window_title
 
     logging.debug(properties)
 
-    return properties
+    return str(properties)
```

Comment:
Well, this actually breaks on proxy_contents.py which expects context to be a dictionary. So this is an unsolved bug.